src/aihwkit/inference/noise/pcm.py

```diff
@@ -34,7 +39,7 @@ class PCMLikeNoiseModel(BaseNoiseModel):
     r"""Noise model that was fitted and characterized on real PCM devices.
 
     Expected weight noise at assumed time of inference with expected
-    programming noise at 0.
+/dccstor/transformer/charles/aihwkit/src/aihwkit/utils    programming noise at 0.
```

Comment:
This seems like it shouldn't be there.

Reply:
Also, the new examples/notebooks need to be run at least one time before merging this. @PabloCarmona can you please do this? When they are working, you can merge.

Reply:
Also, `FAILED tests/test_simulator_tiles.py::TileForwardBackwardTest_ConstantStep::test_set_forward_out_noise_std` failed. That also needs to pass. Maybe run it again and see if it was just a rounding error.

Reply:
I passed that test three times and OK for me. Also I'm gonna remove that text you pointed out and review the notebook. Thanks!

Reply:
Confirmed that in CUDA enabled envs is passing also.